src/node_crypto.cc

```diff
@@ -208,6 +208,9 @@ static int X509_up_ref(X509* cert) {
   return 1;
 }
 #endif  // !OPENSSL_IS_BORINGSSL
+
+#define EVP_MD_CTX_new EVP_MD_CTX_create
+#define EVP_MD_CTX_free EVP_MD_CTX_destroy
```

Comment:
Same comment as the previous commit.  I'd rather avoid the extra allocation by aliasing `EVP_MD_CTX_reset()`.

Reply:
(Same response; OpenSSL 1.1.0 requires the heap allocation. :-( )